src/backend/distributed/utils/citus_clauses.c

```diff
@@ -27,7 +27,7 @@
 
 
 /* private function declarations */
-static bool IsVarNode(Node *node);
+static bool CannotCitusEvaluate(Node *node);
```

Comment:
What does CannotCitusEvaluate mean?

Reply:
CannotCitusEvaluate returns whether citus_evaluate_expr cannot evaluate node

Reply:
Maybe using CanCitusEvaluateExpression is easier to understand

Reply:
Maybe just `IsVarOrParamSublink`. If we need to add another condition, we can come up with a new name.